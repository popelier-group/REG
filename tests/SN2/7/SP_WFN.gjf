# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.50480400   -0.00006700    0.00021900
 H                 -0.78254200   -0.88509100   -0.55108600
 H                 -0.78260500    0.91989400   -0.49053200
 H                 -0.78241000   -0.03503600    1.04236400
 Br                 1.58676900    0.00000100   -0.00000600
 Cl                -3.34016100    0.00002700   -0.00008300

frame.wfn
