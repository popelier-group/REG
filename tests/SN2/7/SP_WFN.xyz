
6

C       -0.391540    0.009325    0.060150
H       -0.544652   -0.538563    0.977057
H       -0.816937   -0.471662   -0.807046
H       -0.636313    1.057449    0.136806
Br        1.675270   -0.039004   -0.257080
Cl       -3.193372    0.074233    0.490006
