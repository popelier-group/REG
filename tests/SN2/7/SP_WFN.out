 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.7/Gau-18942.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.7/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     18943.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.5048   -0.00007   0.00022 
 H                    -0.78254  -0.88509  -0.55109 
 H                    -0.78261   0.91989  -0.49053 
 H                    -0.78241  -0.03504   1.04236 
 Br                    1.58677   0.       -0.00001 
 Cl                   -3.34016   0.00003  -0.00008 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.504804   -0.000067    0.000219
      2          1           0       -0.782542   -0.885091   -0.551086
      3          1           0       -0.782605    0.919894   -0.490532
      4          1           0       -0.782410   -0.035036    1.042364
      5         35           0        1.586769    0.000001   -0.000006
      6         17           0       -3.340161    0.000027   -0.000083
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.079047   0.000000
     3  H    1.079045   1.806000   0.000000
     4  H    1.079052   1.806011   1.806007   0.000000
     5  Br   2.091573   2.588573   2.588581   2.588585   0.000000
     6  Cl   2.835357   2.761965   2.761845   2.762248   4.926930
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.391540    0.009325    0.060150
      2          1           0       -0.544652   -0.538563    0.977057
      3          1           0       -0.816937   -0.471662   -0.807046
      4          1           0       -0.636313    1.057449    0.136806
      5         35           0        1.675270   -0.039004   -0.257080
      6         17           0       -3.193372    0.074233    0.490006
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         153.7423220           0.8488166           0.8488166
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       177.0161742703 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  8.79D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 EnCoef did   100 forward-backward iterations
 SCF Done:  E(RB3LYP) =  -3074.39348314     A.U. after   11 cycles
            NFock= 11  Conv=0.95D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.67693-101.26417 -62.32992 -56.14965 -56.14666
 Alpha  occ. eigenvalues --  -56.14666 -10.07862  -9.17747  -8.53916  -6.93632
 Alpha  occ. eigenvalues --   -6.93544  -6.93544  -6.36849  -6.35864  -6.35864
 Alpha  occ. eigenvalues --   -2.47969  -2.47693  -2.47693  -2.46912  -2.46912
 Alpha  occ. eigenvalues --   -0.64589  -0.54052  -0.51286  -0.29810  -0.29810
 Alpha  occ. eigenvalues --   -0.22759  -0.14847  -0.14847  -0.06440  -0.06372
 Alpha  occ. eigenvalues --   -0.06372
 Alpha virt. eigenvalues --    0.09988   0.11142   0.14356   0.14356   0.16041
 Alpha virt. eigenvalues --    0.17535   0.18132   0.18133   0.21307   0.22029
 Alpha virt. eigenvalues --    0.22029   0.24225   0.24225   0.24776   0.31397
 Alpha virt. eigenvalues --    0.31398   0.31705   0.34655   0.40502   0.50624
 Alpha virt. eigenvalues --    0.52929   0.52930   0.54491   0.54491   0.66369
 Alpha virt. eigenvalues --    0.67320   0.67321   0.69358   0.75229   0.75230
 Alpha virt. eigenvalues --    0.75784   0.78097   0.78097   0.83287   0.83287
 Alpha virt. eigenvalues --    0.84702   0.94827   1.09165   1.14157   1.14158
 Alpha virt. eigenvalues --    1.14673   1.14673   1.16006   1.23274   1.23275
 Alpha virt. eigenvalues --    1.28777   1.56146   1.67075   1.67075   1.79419
 Alpha virt. eigenvalues --    1.86419   1.86419   2.06568   2.13115   2.13116
 Alpha virt. eigenvalues --    2.29586   2.29871   2.29871   2.51880   2.51882
 Alpha virt. eigenvalues --    2.57531   2.61643   2.64427   2.64428   2.70371
 Alpha virt. eigenvalues --    2.70371   3.10579   3.18089   3.85791   3.85792
 Alpha virt. eigenvalues --    4.28843   4.28843   4.36987   4.36987   4.64556
 Alpha virt. eigenvalues --    6.73836   7.70185   7.70185   7.80798   9.88244
 Alpha virt. eigenvalues --   24.04569  26.08001  26.08003  26.23668  48.03387
 Alpha virt. eigenvalues --  215.85163 289.90200 289.90201 290.021441020.81112
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.065487   0.413677   0.413686   0.413653   0.087249  -0.064931
     2  H    0.413677   0.528182  -0.034890  -0.034884  -0.054680   0.002843
     3  H    0.413686  -0.034890   0.528167  -0.034886  -0.054678   0.002842
     4  H    0.413653  -0.034884  -0.034886   0.528218  -0.054679   0.002847
     5  Br   0.087249  -0.054680  -0.054678  -0.054679  35.423307   0.008259
     6  Cl  -0.064931   0.002843   0.002842   0.002847   0.008259  17.903785
 Mulliken charges:
               1
     1  C   -0.328820
     2  H    0.179752
     3  H    0.179759
     4  H    0.179731
     5  Br  -0.354778
     6  Cl  -0.855644
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.210422
     5  Br  -0.354778
     6  Cl  -0.855644
 Electronic spatial extent (au):  <R**2>=           1148.5665
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              9.5296    Y=             -0.2212    Z=             -1.4622  Tot=              9.6437
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -93.4549   YY=            -43.8058   ZZ=            -44.9485
   XY=              1.1548   XZ=              7.6225   YZ=             -0.1772
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -32.7185   YY=             16.9306   ZZ=             15.7879
   XY=              1.1548   XZ=              7.6225   YZ=             -0.1772
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=            207.0604  YYY=             -0.6038  ZZZ=            -10.5423  XYY=             21.7053
  XXY=             -3.8308  XXZ=            -25.1551  XZZ=             24.9083  YZZ=             -1.4650
  YYZ=             -3.2284  XYZ=              0.3742
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1885.4055 YYYY=            -70.7277 ZZZZ=           -102.9749 XXXY=             26.1166
 XXXZ=            172.1194 YYYX=             15.5717 YYYZ=             -2.3848 ZZZX=            108.0525
 ZZZY=             -2.7706 XXYY=           -255.0103 XXZZ=           -270.0841 YYZZ=            -28.9273
 XXYZ=             -2.1844 YYXZ=             35.4610 ZZXY=              6.1450
 N-N= 1.770161742703D+02 E-N=-7.705655769179D+03  KE= 3.070000563523D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE557\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-0.504804,-0.000067,0.000219\H,0,-0.782542,-0.885091,-0.551086\H,0
 ,-0.782605,0.919894,-0.490532\H,0,-0.78241,-0.035036,1.042364\Br,0,1.5
 86769,0.000001,-0.000006\Cl,0,-3.340161,0.000027,-0.000083\\Version=ES
 64L-G16RevC.01\State=1-A\HF=-3074.3934831\RMSD=9.505e-09\Dipole=3.7941
 099,-0.0000743,0.0002409\Quadrupole=-25.2149492,12.6074888,12.6074603,
 0.0003259,-0.0010577,-0.0000032\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 A MAN SHOULD NEVER BE ASHAMED TO OWN HE HAS BEEN IN THE WRONG
 WHICH IS BUT SAYING IN OTHER WORDS,
 THAT HE IS WISER TODAY THAN HE WAS YESTERDAY.

                          -- JONATHAN SWIFT
 Job cpu time:       0 days  0 hours  0 minutes 32.4 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 17.6 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:34:10 2020.
